### PR TITLE
Add test for model packs in MRVA to test plan

### PR DESCRIPTION
**Pull request by koesie10**
This adds a test for a model pack in MRVA that should always work and is independent of changes in the CodeQL libraries (unless the actual `sinkModel` is changed). Unfortunately, that does involve creating a few files manually.

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

**Review by robertbrignull — COMMENTED** [review]
LGTM once the released CLI supports this

**Comment by koesie10** [comment]
v2.16.3 of the CLI now supports this, so this is ready to be merged (once it's been approved).

**Review by robertbrignull — APPROVED** [review]
:shipit:

**Pull request merged**
